frontend/test/acceptance/JoinSupporterSpec.scala

```diff
@@ -72,35 +73,115 @@ class JoinSupporterSpec extends FeatureSpec with Browser
       enterDetails.clickContinue()
 
       And("click 'Pay' button,")
-      enterDetails.pay()
+      enterDetails.payStripe()
 
       Then("the Stripe Checkout iframe should display")
       assert(enterDetails.stripeCheckoutHasLoaded())
 
-      When("the checkout iframe is present")
+      When("The checkout iframe is present")
       enterDetails.switchToStripe()
 
       Then("credit card field is present")
       assert(enterDetails.stripeCheckoutHasCC())
 
-      Then("expiry date field is present")
+      And("expiry date field is present")
       assert(enterDetails.stripeCheckoutHasExph())
 
-      Then("CVC field is present")
+      And("CVC field is present")
       assert(enterDetails.stripeCheckoutHasCVC())
 
-      Then("submit button is present")
+      And("submit button is present")
       assert(enterDetails.stripeCheckoutHasSubmit())
 
-      And("they fill in credit card payment details,")
-      enterDetails.fillInCreditCardPaymentDetailsStripe()
+      When("They fill in credit card payment details,")
+      enterDetails.fillInCreditCardPaymentDetailsStripe
+
+      And("accept the payment")
+      enterDetails.acceptStripePayment
+
+      Then("they should land on 'Thank You' page,")
+      assert(ThankYou.pageHasLoaded)
+
+      And("should be signed in as Supporter.")
+      assert(ThankYou.userIsSignedInAsSupporter)
+    }
+
+    scenario("User joins as Supporter using PayPal", Acceptance) {
+
+      checkDependenciesAreAvailable
+
+      val testUser = new TestUser
+
+      Given("users click 'Become a Supporter' button on Membership homepage")
+
+      When("They land on 'Identity Frontend' page,")
+      val register = pages.Register(testUser)
+      go.to(register)
+      assert(register.pageHasLoaded)
+
+      And("fill in personal details,")
+      register.fillInPersonalDetails()
+
+      And("submit the form to create their Identity account,")
+      register.submit()
+
+      Then("they should land on 'Enter Details' page,")
+      val enterDetails = pages.EnterDetails(testUser)
+      assert(enterDetails.pageHasLoaded)
+
+      When("We switch to the PayPal AB variant")
+      enterDetails.switchToPayPalVariant
+
+      Then("they should have Identity cookies,")
+      Seq("GU_U", "SC_GU_U", "SC_GU_LA").foreach { idCookie =>
+        assert(Driver.cookiesSet.map(_.getName).contains(idCookie))
+      }
+
+      And("should be logged in with their Identity account.")
+      assert(enterDetails.userIsSignedIn)
+
+      When("Users select USA delivery country,")
+      enterDetails.changeCountry("US")
+
+      Then("the currency should change.")
+      assert(enterDetails.currencyHasChanged)
```

Comment:
[We do](https://github.com/guardian/membership-frontend/pull/1517/files#diff-7f40b1cfbd6356bb7297b4ca0f47f200R20) 🙂